ui/components/app/confirm-data/index.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import PropTypes from 'prop-types';
```

Comment:
I would prefer that we don't use 'index.js' like this, it makes finding the file via commands harder, makes quickly grepping vscode tabs harder, etc. For example the transaction controller is currently one of the most difficult files to quickly access from muscle memory commands because its primary code is in controllers/transaction/index.js

Reply:
I updated the PR